src/main/java/org/alfresco/bm/rest/CustomSearchEventWeight.java

```diff
@@ -0,0 +1,59 @@
+package org.alfresco.bm.rest;
+
+import static org.alfresco.bm.rest.SiteContextConstants.EVENT_INSIGHT_ENGINE;
+import static org.alfresco.bm.rest.SiteContextConstants.EVENT_SEARCH_SERVICES;
+import static org.alfresco.bm.rest.SiteContextConstants.SEARCH_SERVICES_SEARCH_TYPE;
+import static org.alfresco.bm.rest.SiteContextConstants.INSIGHT_ENGINE_SEARCH_TYPE;
+
+import org.alfresco.bm.driver.event.EventWeight;
+
+/**
+ * Data representing relative weight for given event name.
+ * 
+ * @author Cristina Diaconu
+ *
+ */
+public class CustomSearchEventWeight extends EventWeight {
+
+	private String searchType;
```

Comment:
You've used tabs throughout the java files. Is that the standard for this project?  Usually Alfresco source code is indented with spaces.

Reply:
I've updated the file using Java Conventions Alfresco formatter.